k8s/scripts/inspect.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+INSPECT_DUMP=$(pwd)/inspection-report
+
+function log_success {
+  printf -- '\033[32m SUCCESS: \033[0m %s\n' "$1"
+}
+
+function log_info {
+  printf -- '\033[34m INFO: \033[0m %s\n' "$1"
+}
+
+function log_warning () {
+  printf -- '\033[33m WARNING: \033[0m %s\n' "$1"
+}
+
+function log_warning_red {
+  printf -- '\033[31m WARNING: \033[0m %s\n' "$1"
+}
+
+function is_control_plane_node {
+  k8s local-node-status | grep -q "control-plane"
+}
+
+function collect_args {
+  log_info "Copy service args to the final report tarball"
+  cp -r --no-preserve=mode,ownership /var/snap/k8s/common/args "$INSPECT_DUMP"
+}
+
+function collect_cluster_info {
+  log_info "Copy k8s cluster-info dump to the final report tarball"
+  k8s kubectl cluster-info dump &> "$INSPECT_DUMP/cluster-info.log"
```

Comment:
let's do `k8s kubectl cluster-info dump --output-directory` instead, as that has a much better structure than the single file